src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -167,12 +171,17 @@ def _create_all_cardinalities(class_name: str, class_df_with_cardinalities: pd.D
 
 def _check_complete_gui_order(class_name: str, class_df_with_cardinalities: pd.DataFrame) -> pd.DataFrame:
     detail_problem_msg = ""
+    attempt_conversion = False
     if "gui_order" not in class_df_with_cardinalities:
-        detail_problem_msg = "the column 'gui_order' does not exist."
+        pass
+    elif class_df_with_cardinalities["gui_order"].isna().all():
+        pass
     elif class_df_with_cardinalities["gui_order"].isna().any():
         detail_problem_msg = "some rows in the column 'gui_order' are empty."
+    elif not class_df_with_cardinalities["gui_order"].isna().all():
```

Comment:
technically, it would be more correct to turn this into an `else`. But I understand that sometimes, for the human brain it is more helpful to have another elif.

Reply:
yes, but then I would have to do another if clause in the else, I don't like to do that...

Reply:
No, you don't need an if clause. The exact opposite of this if clause is already tested on line 177, so line 181 will always return True.

Reply:
I meant the wrong elif, but yes, I preferred it for expressiveness...